### PR TITLE
DAOS-4332 test: Adding crashior.py test

### PR DESCRIPTION
**Pull request by saurabhtandan**
As part of these changes a test is added to
weekly run to kill an ongoing io process
(achieved using ior in this case) and check
for server status and make sure no server
crashed.
In order to achieve this some utilities have
also been added to existing files.

Test-tag-hw-medium:pr,hw,medium,ib2 crashior

Signed-off-by: Saurabh Tandan <saurabh.tandan@intel.com>

### CODE REVIEW

File: src/tests/ftest/util/command_utils.py [hunk 1]
```diff
@@ -223,6 +223,27 @@ def stop(self):
             self.log.info("%s stopped successfully", self.command)
             self._process = None
 
+    def wait(self):
+        """Wait for the sub process to complete.
+
+        Args:
+            timeout (int): number of seconds to wait for the sub process
```
Comment (shimizukko) [inline src/tests/ftest/util/command_utils.py:230]: Do we have this argument?

Comment (saurabhtandan) [reply src/tests/ftest/util/command_utils.py]: Done

File: src/tests/ftest/util/dmg_utils.py [hunk 2]
```diff
@@ -816,6 +817,52 @@ def pool_set_prop(self, pool, name, value):
         self.sub_command_class.sub_command_class.value.value = value
         return self._get_result()
 
+    def pool_exclude(self, pool_uuid, rank, tgt_idx=None):
+        """Exclude a daos_server from the pool
+
+        Args:
+            pool (str): Pool uuid.
+            rank (int): Rank of the daos_server to exclude
+            tgt_idx (int): target to be excluded from the pool
```
Comment (shimizukko) [inline src/tests/ftest/util/dmg_utils.py]:
```suggestion
            tgt_idx (int, optional): target to be excluded from the pool. Defaults to None.
```

File: src/tests/ftest/util/dmg_utils.py [hunk 3]
```diff
@@ -816,6 +817,52 @@ def pool_set_prop(self, pool, name, value):
...
+        self.sub_command_class.sub_command_class.tgt_idx.value = tgt_idx
+        return self._get_result()
+
+    def pool_reintegrate(self, pool_uuid, rank, tgt_idx=None):
+        """Reintegrate a daos_server to the pool
+
+        Args:
+            pool (str): Pool uuid.
+            rank (int): Rank of the daos_server to reintegrate
+            tgt_idx (int): target to be reintegrated to the pool
```
Comment (shimizukko) [inline src/tests/ftest/util/dmg_utils.py]:
```suggestion
            tgt_idx (int, optional): target to be reintegrated to the pool. Defaults to None.
```

**Comment by saurabhtandan** [comment]
Rebuild test and nvme_object.py test failures are not related to changes made as part of this PR

**Pull request merged**